pkg/config/piped.go

```diff
@@ -218,12 +218,16 @@ func (s *PipedSpec) FindPlatformProvider(name string, t model.ApplicationKind) (
 	return PipedPlatformProvider{}, false
 }
 
-// FindPlatformProvidersByLabel finds all PlatformProviders which match the provided labels.
-func (s *PipedSpec) FindPlatformProvidersByLabel(labels map[string]string, t model.ApplicationKind) []PipedPlatformProvider {
+// FindPlatformProvidersByLabels finds all PlatformProviders which match the provided labels.
+func (s *PipedSpec) FindPlatformProvidersByLabels(labels map[string]string, t model.ApplicationKind) ([]PipedPlatformProvider, bool) {
```

Comment:
Do we really need the `found` return value?
I feel that only slice is enough and closer with Go conventions. 🤔

Reply:
I feel you, want to keep this function signature as same as FindPlatformProviderByName 👍 Lets me remove it

Reply:
👍

Reply:
Addressed by https://github.com/pipe-cd/pipecd/pull/3856/commits/3be3f14d28237b6dc3f50babfd8d5f74757e1169